src/main/resources/db/migration/V1__Create_tables.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE soknader (
+    soknad_id VARCHAR(64) NOT NULL,
+    innsendt_dato TIMESTAMP NOT NULL,
+    soknad JSONB,
+    PRIMARY KEY (soknad_id,innsendt_dato)
```

Comment:
Jeg tror egentlig du vil ha en intern id som primærnøkkel siden du kan ha flere innslag på samme søknadsId. Tror jeg ville brukt en vanlig teller. Eller hva database @andersostby ?

Reply:
Bør ikke (soknad_id,innsendt_dato) være unik? (innsendt_dato er et dårlig feltnavn ser jeg nå, for det er faktisk en timestamp)

Reply:
Du bruker `opprettet`-feltet fra søknaden. Det beskriver når søknaden ble opprettet i våre systemer. Da vil du kunne ha to innslag på topic:

```
NY_SOKNAD { id: 1, opprettet: 01-01-01 }
SOKNAD_SENDT { id: 1, opprettet: 01-01-01 } 
```

Kanskje du skal innføre et felt: `NY | SENDT | KORRIGERT` som beskriver hvilken hendelse du har behandlet? Så kan du bruke (hendelse, soknadsId) som primærnøkkel?

Reply:
Men er det unikt? Kan man ikke ha flere `KORRIGERT`-hendelser?